docs/sdks/react-native/release-notes.md

```diff
@@ -25,12 +25,20 @@ platforms.
 
 #### ID
 
-* ID Capture supports scanning the Visual Inspection Zone (VIZ) of New York City IDs.
+* ID Capture now supports scanning:
+  * Visual Inspection Zone (VIZ) of New York City IDs.
+  * Florida Medical Marijuana IDs.
 
 #### Label
 
 * Added support for text fields that are identified only based on their data pattern. These do not need textual identifiers to be identified, and therefore no data type patterns (i.e., patterns for the textual identifiers) need to be specified for them.
 
+### Behavioral Changes
+
+#### Barcode
+
+* AR views (`BarcodeTrackingAdvancedOverlay` and `BarcodeTrackingAdvancedOverlayListener`) can only be displayed without image elements in the custom view. Please contact [Scandit Support](mailto:support@scandit.com) for details.
```

Comment:
@gpene This should not be here anymore, all these changes are made in 6.27.0 to 6.27.2 and are part of 6.28 implicitly (the final 6.27.2 changes will only be in the production version and are not in beta.2 yet). Listing them here as well seems confusing.